spec/models/account_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+
+RSpec.describe Account, type: :model do
+  let!(:account) { FactoryBot.create(:account) }
+  let(:edrpou) { account.edrpou }
+  let(:iban) { account.iban }
+  let(:purpose) { account.purpose }
+
+  context "when valid Factory" do
+    it "has a valid factory" do
+      expect(account).to be_valid
+    end
+  end
+
+  describe 'validations' do
+    context 'when edrpou' do
+      it 'require a edrpou' do
+        account.edrpou = edrpou
+        expect(account).to be_valid
+      end
+
+      it 'reject edrpou not according to the template' do
+        account.edrpou = 'wrong'
+        expect(account).not_to be_valid
+      end
+    end
+
+    context 'when iban' do
+      it 'be valid' do
+        account.iban = iban
+        expect(account).to be_valid
+      end
+
+      it 'reject iban not according to the template' do
+        account.iban = 'wrong'
+        expect(account).not_to be_valid
+      end
+    end
+
+    context 'when purpose' do
```

Comment:
better to move it in describe block, please read differense beetween describe/context blocks in rspec

Reply:
refactor